qgcresources.qrc

```diff
@@ -3,11 +3,8 @@
         <file alias="opensans">resources/fonts/OpenSans-Regular.ttf</file>
         <file alias="opensans-demibold">resources/fonts/OpenSans-Semibold.ttf</file>
     </qresource>
-
```

Comment:
Doesn't really matter to me. I've done this several times before, so this time I just kept the change. Your call.